src/layout/strategy/local.js

```diff
@@ -0,0 +1,83 @@
+import d3 from 'd3';
+import {rebindAll} from '../../util/rebind';
+import minimum from '../../util/minimum';
+import {allWithCollisions, areaOfCollisions} from './collision';
+import containerUtils from './container';
+import {getAllPlacements} from './placement';
+
+export default function() {
+
+    var container = containerUtils();
+    var iterations = 1;
+
+    var strategy = function(data) {
+
+        var originalData = data;
+        var iteratedData = data;
+
+        var lastIterationScore = Infinity;
+        for (var i = 0; i < iterations; i++) {
+            iteratedData = iterate(originalData, iteratedData);
+
+            // Reached a local minimum?
+            var thisIterationScore = areaOfCollisions(iteratedData);
+            if (lastIterationScore === thisIterationScore) {
+                return iteratedData;
```

Comment:
Rather than an early-return, I think this would be more clearly expressed as a while loop, i.e. repeat while i < iterations AND score < previousScore